package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "react_global_training",
+  "version": "1.0.0",
+  "description": "react global training (EPAM)",
+  "main": "./src/index.js",
+  "scripts": {
+    "start": "webpack serve",
+    "build": "SET NODE_ENV=production & webpack",
```

Comment:
it's better to use [cross-env](https://www.npmjs.com/package/cross-env), cause it will not set `NODE_ENV` for mac machine for example.
```js
"build": "cross-env NODE_ENV=production webpack",
```

Reply:
Да спасибо, уже в след таске его использую